sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveTableScanSuite.scala

```diff
@@ -187,6 +188,35 @@ class HiveTableScanSuite extends HiveComparisonTest with SQLTestUtils with TestH
     }
   }
 
+  test("HiveTableScanExec should not increase data parallelism") {
```

Comment:
`should not increase data parallelism` -> `should respect HIVE_TABLE_SCAN_MAX_PARALLELISM`?